kyaml/kio/ignorefilesmatcher.go

```diff
@@ -30,14 +30,14 @@ import (
 //   package contains a pattern that ignores the directory foo, if foo is a
 //   subpackage, it will still be included if the IncludeSubpackages property
 //   is set to true
-type ignoreFilesMatcher struct {
+type IgnoreFilesMatcher struct {
```

Comment:
Why is this needed? The comment should be updated with the new name of the type.